lib/jetstream/api/stream.ex

```diff
@@ -31,6 +40,130 @@ defmodule Jetstream.API.Stream do
 
   @type nanoseconds :: non_neg_integer()
 
+  @typedoc """
+  Stream type fields explanation:
+
+  * `:allow_rollup_hdrs`: allows the use of the Nats-Rollup header to replace all contents of a stream,
```

Comment:
Ecto documents struct fields in module doc: https://hexdocs.pm/ecto/Ecto.Changeset.html#module-the-ecto-changeset-struct. I think that approach may be more discoverable when browsing the docs. Wdyt?